common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -290,3 +290,9 @@ def set_wrong_inputs_to_fields(self):
                 "Course Announcement Date": '"string"',
             }
         )
+
+    def test_hidden_fields_not_displayed(self):
```

Comment:
That sounds like a good idea to me -- i'll work on it and add to the PR

On Thu, Jan 22, 2015 at 1:26 PM, Christina Roberts <notifications@github.com
> wrote:

> In common/test/acceptance/tests/studio/test_studio_settings.py
> <https://github.com/edx/edx-platform/pull/6718#discussion_r23397074>:
>
> > @@ -290,3 +290,9 @@ def set_wrong_inputs_to_fields(self):
> >                  "Course Announcement Date": '"string"',
> >              }
> >          )
> > +
> > +    def test_hidden_fields_not_displayed(self):
>
> I'm not sure this test is really useful (since you have to update the code
> 2 places, and it won't catch things that you forgot to filter out).
>
> I think what would be more useful is a test that checks for the elements
> that ARE present (and verifies no extra settings are present). That way, if
> you add something new, the test fails and you have to think about whether
> or not it should be exposed in Advanced Settings. If it should be, you
> update the test. This would have caught a number of items that
> inadvertently showed up in Advanced Settings.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/6718/files#r23397074>.
>



-- 
Matthew Drayer
Solutions Architect
edX | MIT
mattdrayer@edx.org